compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -158,16 +148,15 @@ class Context(val contextId: Context.ContextId) {
 
   private def buildMachine(identifier: Identifier): Option[Speedy.Machine] = {
     for {
-      res <- defns.get(LfDefRef(identifier))
-      (lfVer, defn) = res
+      defn <- defns.get(LfDefRef(identifier))
     } yield
     // note that the use of `Map#mapValues` here is intentional: we lazily project the
     // definition out rather than rebuilding the map.
     Speedy.Machine
       .build(
-        checkSubmitterInMaintainers = VersionTimeline.checkSubmitterInMaintainers(lfVer),
+        checkSubmitterInMaintainers = false,
```

Comment:
I assume you are planning to remove this check in a follow-up PR?

Reply:
Yes thanks. everything related to this will go away. just piece by piece :)

Reply:
Yes.